examples/gateway/azure_openai.yaml

```diff
@@ -0,0 +1,36 @@
+routes:
+  - name: chat
+    route_type: llm/v1/chat
+    model:
+      provider: openai
+      name: gpt-3.5-turbo
+      config:
+        openai_api_type: "azure"
+        openai_api_key: $OPENAI_API_KEY
+        openai_deployment_name: "{your_deployment_name}"
+        openai_api_base: "https://{your_resource_name}-azureopenai.openai.azure.com/"
+        openai_api_version: "2023-05-15"
+
+  - name: completions
+    route_type: llm/v1/completions
+    model:
+      provider: openai
+      name: gpt-3.5-turbo
```

Comment:
I think MSFT omits the `.` in this model name https://learn.microsoft.com/en-gb/azure/cognitive-services/openai/concepts/models#chatgpt-gpt-35-turbo

Reply:
Thanks Ben! Fixed!